index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="header">
+        <div class="logo">
+            <img src="images/logo.jpg" alt="logo"/>
+        </div>
+        <div class="user">
+            <a href="#">John Snow</a>
+        </div>
+    </div>
+    <div class="sidebar">
+        <div class="plan-button">
+                <form>
+                    <select class="plans-list">
+                        <option value="plan">Select Plan</option>
+                        <option value="blueberry">Blueberry Plan</option>
+                        <option value="Strawberry">Strawberry Plan</option>
+                        <option value="Apple">Apple</option>
+                        <option value="Mango">Mango</option>
+                    </select>
+                </form> 
+        </div>
+        <div class="plan-info">
+            <div class="location">
+                <a href=""><img src="images/location.png" alt="location"/> </a>
+                <ul class="location-list">
+                    <li class="location-one current">
+                        <a href="#"><img src="images/location-list.png" alt=""></a>
+                        <span>Store</span>
+                    </li>
+                    <li class="location-one store">
+                        <a href="#"><img src="images/location-list.png" alt=""></a>
+                        <span>Transport</span>
+                    </li>
+                    <li class="location-one store">
+                        <a href="#"><img src="images/location-list.png" alt=""></a>
+                        <span>Sell</span>
+                    </li>
+                    <li class="location-one store">
+                        <a href="#"><img src="images/location-list.png" alt=""></a>
+                        <span>Use</span>
+                    </li>
+                </ul>               
+            </div>
+            <p>Make</p>
+            <div class="detector"><img src="images/detector.png" alt=""/></div>
+            <div class="about-plan">
+                <p>Plan Info</p><br><br><br>
+                <pre>Point Used    :4.5 pts</pre>
```

Comment:
@priyanka1110 with using pre tag the way we write the content with space in between in html, browser will show as it is which is not possible with the p tag, where p tag will be called in css and given letter spacing. This can be done with pre tag